src/tests/Microsoft.PowerFx.Core.Tests/DNameTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.Utils;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public sealed class DNameTests
+    {
+        [Fact]
```

Comment:
Can this use [Theory] and [InlineData]? Separates out each test case individually, makes it easier to debug in VS